archetypes/blog-post/index.md

```diff
@@ -0,0 +1,42 @@
+---
+title: "{{ replace .Name "-" " " | title }}"
+date: {{ .Date }}
+publishDate: {{ .Date }}
+draft: true
+meta_image: meta.png
```

Comment:
And maybe another test that ensures the placeholder image doesn't exist at all for the post, to ensure it's been deleted.

Perhaps some of these test would only be run if `hugo.Environment` is not `development`, so the build wouldn't fail during development when you're just getting started writing the post and haven't swapped in a new image yet, but would fail in the environments we want them to fail like staging and production.

Reply:
I do like that, yeah. Good suggestions, I'll adjust. 💤

Reply:
Is there a way to test if the image is the correct size? They should be 2:1 in dimension and we've been doing 1200x600.